GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -2,19 +2,20 @@
 using GVFS.Common.FileSystem;
 using GVFS.Common.Tracing;
 using GVFS.Platform.POSIX;
-
+using System.IO;
+
 namespace GVFS.Platform.Mac
 {
     public partial class MacPlatform : POSIXPlatform
     {
-        public MacPlatform() : base(installerExtension: ".dmg")
+        public MacPlatform()
         {
         }
 
         public override IDiskLayoutUpgradeData DiskLayoutUpgrade { get; } = new MacDiskLayoutUpgradeData();
         public override IKernelDriver KernelDriver { get; } = new ProjFSKext();
         public override string Name { get => "macOS"; }
-
+        public override GVFSPlatformConstants Constants { get; } = new MacPlatformConstants();

```

Comment:
Please put the initialization inside the constructor.

Reply:
As discussed, this can be done in a follow-up PR. Created a [tracking issue](https://github.com/Microsoft/VFSForGit/issues/1083).